Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py

```diff
@@ -1028,16 +1028,20 @@ def get_new_indicators(
 
         param_start_date: datetime = datetime.fromtimestamp(0)
         if start_date is not None:
+            if start_date.tzinfo is None:
+                start_date = pytz.utc.localize(start_date)
+            else:
+                start_date = start_date.astimezone(pytz.UTC)
             param_start_date = max(
-                earliest_fetch.replace(tzinfo=pytz.UTC), start_date.replace(tzinfo=pytz.UTC)
+                earliest_fetch.replace(tzinfo=pytz.UTC), start_date
```

Comment:
is there is a logic different between this code and `max(param_start_date = max(
                earliest_fetch.replace(tzinfo=pytz.UTC), start_date.replace(tzinfo=pytz.UTC)`
?